README.debian.org

```diff
@@ -0,0 +1,906 @@
+#+TITLE: Gerbil in .deb packages
+
+Gerbil. I use it for production servers. With VM's, often, Ubuntu is the way. A
+package is the way to go, with an apt repository so it's automagically
+distributed. Thus, here we go.
+
```

Comment:
> So what tags do you need? We auto-tag using the `release.ss` script.

Debian needs an upstream tarball, and a git tag that points to the version. I generate that, and rather than add patches to debian/ with all my Makefile f*ckery, I made a v0.15.2 tag. 

https://github.com/drewc/gerbil/tree/v0.15.2

Then, the debian/* tree is added, and I tagged the package name. v0.15.2-1ppa2

https://github.com/drewc/gerbil/tree/v0.15.2-1ppa2

That way I can simply checkout if/when I need to develop an upgraded package. For example, I plan to make a gerbil-snapshot package that simply pulls from master. That will be a tag as well.

The idea is to keep everything on one branch (debian), and tag packages where needed (different dists, CPUs etc). The way I see it is that tags are plenty, branches should be minimal. 

I wanted to know where our versioning comes from, thank you for mentioning release.ss! :)

Reply:
So what happens when we tag the v0.15.2 release for good in master?
(Note: that will probably be when gambit v4.9.4 is released, but can do it sooner)

Reply:
> This file looks redundant with the files in `debian/` — or are the latter files extracted from this one via e.g. noweb? If so, this deserves a prominent mention at the top of each and every generated file (if comments are allowed).

You've got it, it's a literate README, and all code is tangled from that. I should mention so? Probably a good idea.

I developed an "untangle" about a decade ago in CL. It took source files, tangled a temporary tree, ran a cl-diff, and then computed what lines came from what code block. Then, it re-wrote the code blocks and re-tangled. No diff anymore. 

I have plans to do so in gerbil shortly.

Reply:
So is the `debian/` directory generated from `README.debian.org` or manually written independently? Either case should be made clearer in the code and the README.